lmnet/lmnet/datasets/dataset_iterator.py

```diff
@@ -206,10 +210,18 @@ def __init__(self, dataset, enable_prefetch=False, seed=0):
         self.dataset = dataset
         self.enable_prefetch = enable_prefetch
         self.seed = seed
+        queue_size = dataset.queue_size if dataset.queue_size else 50
```

Comment:
`queue_size` and `num_parallel_process` are only for `DatasetIterator` and `_MultiProcessDatasetPrefetchThread`, `Dataset` class shouldn't have these `queue_size` and `num_parallel_process` parameter 🤔🤔🤔 
Could you change these to according to encapsulation ways.

Reply:
@ruimashita 
Sorry, I was confused by the term `encapsulation` and where should I use it. Anyway, I've fixed as suggested by adding parameters in the DatasetIterator.__init__()

Reply:
@ananno 
Oh, No. You are not wrong. Sorry for my poor description 😭😭 
Thanks for contribution